app/assets/javascripts/discourse/app/components/d-editor.js

```diff
@@ -546,10 +546,15 @@ export default Component.extend(TextareaTextManipulation, {
 
           // note this will only work for emojis starting with :
           // eg: :-)
+          let emojiTranslation = this.get("site.custom_emoji_translation");
```

Comment:
Can we use something like `this.get("site.custom_emoji_translation") || {}`? Would take fewer lines of code. 

Also, you can use `const emojiTranslation` in that case.